### PR TITLE
added week and quarter to persian/farsi

### PR DESCRIPTION
**Pull request by Crimson-Amir**
There was no equivalent of week and quarter in Persian language.
I use this library and I had to use try-except to handle the week and this made me do this contribute.

### CODE REVIEW

**Review by jadchaar — APPROVED** [review]

**Comment by jadchaar** [comment]
Thanks @Crimson-Amir for the contribution. Mind adding some unit tests?

**Review by jadchaar — APPROVED** [review]

**Comment by jadchaar** [comment]
@Crimson-Amir would it be possible to add some unit tests? Then we can merge this in.

**Comment by Crimson-Amir** [comment]
Sure

**Comment by Crimson-Amir** [comment]
can you check one more time? It doesn't seem like a big problem

**Comment by jadchaar** [comment]
Hey @Crimson-Amir thanks for adding tests. Mind running `tox -e lint` to fix the linting issues?

File: tests/test_locales.py [hunk 1]
```diff
@@ -1443,6 +1443,27 @@ def test_timeframes(self):
...
+        assert self.locale._format_timeframe("week", 1) == "یک هفته"
+        assert self.locale._format_timeframe("month", 1) == "یک ماه"
+        assert self.locale._format_timeframe("year", 1) == "یک سال"
+
+        # double
+        assert self.locale._format_timeframe("minutes", 2) == "2 دقیقه"
+        assert self.locale._format_timeframe("hours", 2) == "2 ساعت"
+        assert self.locale._format_timeframe("days", 2) == "2 روز"
+        assert self.locale._format_timeframe("weeks", 2) == "2 هفته"
+        assert self.locale._format_timeframe("months", 2) == "2 ماه"
```
Comment (krisfremen) [inline tests/test_locales.py:1463]: missing tests for quarters, otherwise 👍

**Review by krisfremen — APPROVED** [review]
quarter tests are missing, but LGTM otherwise

**Pull request merged**